src/app/content/components/utils/applySearchIconColor.ts

```diff
@@ -0,0 +1,28 @@
+import { css } from 'styled-components/macro';
+import theme from '../../../theme';
+import { BookWithOSWebData } from '../../types';
+import { toolbarIconColor } from '../constants';
+
+export const applySearchIconColor = (props: {colorSchema: BookWithOSWebData['theme']}) =>
+  props.colorSchema ? css`
```

Comment:
unless they recently fixed this the arguments to the functions you call in a styled-component block don't have strong typing, so i prefer not to hide them in helpers. for instance here you might make the function accept just the `schema` and do the prop extraction in the styled block